src/components/Item/VisualizationItem/ItemHeaderButtons.js

```diff
@@ -105,18 +111,23 @@ const ItemHeaderButtons = (props, context) => {
         </>
     )
 
+    const buttonRef = createRef()
+
     return pluginIsAvailable(activeType || item.type) ? (
         <>
-            <Button small secondary onClick={handleMenuClick}>
-                <ThreeDots />
-            </Button>
-            {anchorEl && (
+            <div ref={buttonRef}>
+                <Button small secondary onClick={handleMenuClick}>
```

Comment:
@cooper-joe since now the button is always visible, should it toggle the menu?

Reply:
actually it might already do that because of the `onClickOutside` prop in `Popover`...